fabio/edfimage.py

```diff
@@ -287,13 +369,18 @@ def _extract_header_metadata(self, capsHeader=None):
             else:
                 self._data_swap_needed = False
 
-    def parseheader(self, block):
+    #PB38k20190607:
+    # This function has been renamed from parseheader to _parseheader,
+    # because it returns the capitalized header and not the
+    # blobsize, which should be taken from _read_header_block or
+    # if not returned there explicitely from capsHeader after
+    # calling _parseheader.
+    def _parseheader(self, block):
```

Comment:
Those comments have nothing to do in the source code. It should be explained in the commit message.